api/v1alpha1/releaselink_webhook.go

```diff
@@ -72,3 +84,13 @@ func (r *ReleaseLink) isTargetValid() error {
 
 	return nil
 }
+
+// isAutoReleaseLabelValid throws an error if the auto-release label value is set to anything besides true and false.
+func (r *ReleaseLink) isAutoReleaseLabelValid() error {
+	if _, found := r.GetLabels()[autoReleaseLabel]; found {
+		if r.Labels[autoReleaseLabel] != "true" && r.Labels[autoReleaseLabel] != "false" {
```

Comment:
You are already getting the value in the previous line. Replace `_` with `autoReleaseLabel` so you don't search multiple times.

Reply:
Done